home/models.py

```diff
@@ -122,3 +140,16 @@ class Meta:
 
     def __str__(self):
         return f"{self.question}, Tag : {self.tag}"
+
+    # Input: array of strings that represent name of tags and a question.
+    # The function will go through the array and will add them as a pair - question and tag.
```

Comment:
Why not add this is a regular method in Question? I think it would make more sense then passing `question_id` in here and having to query the DB for it...